Documentation/Signoffs/jdjoyce42-power-signoff.md

```diff
@@ -10,21 +10,36 @@ The purpose of this subsystem is to take the voltage supplied from a wall outlet
 
 ## Constraints
 
-### 1) Standard IEC 60950-1 deals with wired and battery powered devices below 600 V
-Standard IEC 60950-1 deals with wired and battery powered devices below 600 V. This device will make use of wall power so this standard will be followed to eliminate harm from electric shock, fire, and mechanical instability.
+| No. | Constraints                                                           | Origin            |
+| --- | --------------------------------------------------------------------- | ----------------- |
+| 1   | System shall convert wall AC Voltage to 5V DC Voltage         | Design Constraint |
```

Comment:
This is impossible. Based on this constraint you must develop a 5VDC power supply with Vp-p = 0 (perfect DC). That's not possible.